debian/changelog

```diff
@@ -1,3 +1,13 @@
+efl (1.18.2-2) experimental; urgency=medium
+
+  * debian/control:
+    - Use TLS-enabled, canonical links
+    - Add missing dependency for the doc build
+    - Add Uploaders (Nicolas Braud-Santoni & Ross Vandegrift)
+  * debian/rules: Test fixup (XDG_RUNTIME_DIR & HOME)
+
+ -- Nicolas Braud-Santoni <nicolas@braud-santoni.eu>  Fri, 04 Nov 2016 16:12:43 +0100
+
```

Comment:
PS: The changelog was in a separate commit, so you could have dropped that yourself anyhow.